src/schema/partner_show_event.ts

```diff
@@ -58,10 +63,20 @@ const PartnerShowEventType = new GraphQLObjectType<any, ResolverContext>({
     },
     start_at: dateFieldForPartnerShowEvent,
     end_at: dateFieldForPartnerShowEvent,
+    datesAreSameDay: {
+      type: GraphQLBoolean,
+      description: "Returns true if dates are on same day",
+      resolve: ({ start_at, end_at }) => datesAreSameDay(start_at, end_at),
+    },
```

Comment:
Seeing as I can’t find `datesAreSameDay` in the Notion doc _and_ I have some concerns about this implementation across timezones, do we really need to expose this now? (i.e. is there a client that needs this currently?)

Reply:
In Emission there's a component that requires two different designs depending on whether the time is the same day, which would need this logic.

https://app.zeplin.io/project/5bef4c7dae2b6a3eb6a75f4a/screen/5c1ae7415ff90824363b8f42
vs. 
https://app.zeplin.io/project/5bef4c7dae2b6a3eb6a75f4a/screen/5c2e4dd581415cb00682abb9

The design could be pushed back on if we don't want to expose this logic. Currently the logic lives in the client and exposing this field would be in preparation for removing Moment.js from Eigen/Emission.

Reply:
Is there a way to move all of that logic out of the client into MP in a single field, rather than exposing this detail?

Reply:
Yeah, after looking at it again we can just use a single field here, I wont expose this field.